### ISSUE TITLE
Skyrim false cycle detection v0.13.0

### ISSUE THREAD

**Issue by Mincol** [loot/loot#963]
> Failed to sort plugins. Details: Cyclic interaction detected between "RealisticWaterTwo - iNeed.esp" and "ELE_SSE.esp". Back cycle: ELE_SSE.esp, SoS_ELE_Patch.esp, RealisticWaterTwo.esp, RealisticWaterTwo - iNeed.esp

Works with LOOT v0.12.5

There is no cycle. ELE_SSE.esp does not depends on any listed mods.

**Comment by PhoenixTheSage** [comment]
I have the same issue. 
[LOOTDebugLog.txt](https://github.com/loot/loot/files/2066073/LOOTDebugLog.txt)

Works with 12.5 however the new masterlist sorts my loadorder that is totally borked on 12.5 now.
As an example, this was my load order before the new masterlist update: 
[plugins.txt](https://github.com/loot/loot/files/2066079/plugins.txt)
Now this is what it looks like with Revision 609a076:
[plugins.txt](https://github.com/loot/loot/files/2066083/plugins.txt)

**Comment by WrinklyNinja** [comment]
```
[11:24:47.967032] [error]: Failed to sort plugins. Details: Cyclic interaction detected between plugins "Smashed Patch.esp" and "Bashed Patch, 0.esp". Back cycle: Bashed Patch, 0.esp, ASIS-Dependency.esp, Immersive Citizens - AI Overhaul.esp, CFTO.esp, CFTO-JK-Patch.esp, Smashed Patch.esp
[11:24:47.997112] [error]: Chromium console message from http://loot/ui/app.bundle.js at line 1: Error: Failed to sort plugins. Details: Cyclic interaction detected between "Smashed Patch.esp" and "Bashed Patch, 0.esp". Back cycle: Bashed Patch, 0.esp, ASIS-Dependency.esp, Immersive Citizens - AI Overhaul.esp, CFTO.esp, CFTO-JK-Patch.esp, Smashed Patch.esp
    at J.e.then.then.then.e 
```

That's not the same issue, please file a separate issue instead of hijacking this one.

**Comment by PhoenixTheSage** [comment]
There is not cyclic dependency there, I've checked, and it most certainly works on 12.5, but not on 13.0 or 13.1 so as far as I could tell it stems from the same issue. But, okay, whatever.

**Comment by WrinklyNinja** [comment]
> There is no cyclic dependency there, I've checked, and it most certainly works on 12.5, but not on 13.0 or 13.1, just because the plugins aren't the same ones doesn't mean it's not the same issue... So as far as I could tell before I posted I thought it was the same issue. But, okay, whatever.

They're the same general issue in the sense that they're both cyclic interactions, but because they involve different plugins, they might need separate solutions, and it's easier to merge duplicate issue threads than it is to keep track of and possibly split up two separate problems in the same issue thread.

**Comment by CptMcSplody** [comment]
@PhoenixTheSage Have you cleared your loot folder in appdata?

**Comment by WrinklyNinja** [comment]
@Mincol Your cycle may be fixed in the latest masterlist, please try updating your masterlist and sorting again.

@CptMcSplody Well, same with the Bashed Patch, we can add an entry for Smashed Patch.esp if the user decides to call it something non-default, it's their responsibility to give it the right metadata.

**Comment by CptMcSplody** [comment]
@WrinklyNinja Okay, I'm double checking with @matortheeternal for its placement. I believe it should be after bashed patch, but I want to be sure.

**Comment by CptMcSplody** [comment]
@Mincol Are you still having the above issue with the latest master-list?

**Comment by Mincol** [comment]
Isse was solved with LOOT v0.13.1 an latest master list.
